beehive/pkg/core/core.go

```diff
@@ -24,7 +24,7 @@ func StartModules() {
 	}
 }
 
-// if get the special signals, cleanup modules
+// GracefulShutdown : if get the special signals, cleanup modules
```

Comment:
GracefulShutdown : -> GracefulShutdown is ?
Applicable to to all similar places